components/admin/data/templates/index.html

```diff
@@ -0,0 +1,8 @@
+{% load socialaccount %}
```

Comment:
This is just to show what `allauth` can do when you log in (it pulls username from GitHub), and can be omitted if we don't want this.